ccmbioinfo/lfs
# CARDS-1728: In PROMs, the print header should display patient name, mrn, dob, and appointment time

**Includes**:
* [CARDS-1727](https://phenotips.atlassian.net/browse/CARDS-1727): The printing header should be configurable via extensions
* [CARDS-1728](https://phenotips.atlassian.net/browse/CARDS-1728): In PROMs, the print header should display patient name, mrn, dob, and appointment time

**Testing**:
* Start the **proms** project,
  * create a patient information form and fill in first/last name, dob, mrn
  * create a visit information form for that patient and fill in visit time
  * create some survey forms, 
  * go to **print preview** both in the survey page and in the visit subject page.
  * **The page header of the print preview should display patient name, dob, MRN, and appointment date/time.**

![image](https://user-images.githubusercontent.com/651980/164524482-021b3cae-a8b7-48bd-b299-467d512d6f9b.png)

* After deleting the data (`rm -rf .cards-data`), start **any other project**,
  * create some forms, 
  * go to **print preview** both in the form and subject page.
  * **The page header of the print preview should display the breadcrumbs and creation date** (unchanged from what was being displayed before this PR).
